API.md

```diff
@@ -1421,7 +1421,7 @@ Alternatives can be expressed using the shorter `[]` notation.
 
 ```javascript
 var alt = Joi.alternatives().try(Joi.number(), Joi.string());
-// Same as [Joi.number(), Joi.string()]
+// Same as Joi.alternatives([Joi.number(), Joi.string()])
```

Comment:
`Alternatives can be expressed using the shorter `[]` notation.`
Add a link to the compile function, it might help.